Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -998,6 +1006,15 @@ class DropDownMenu
     }
   }
 
+  private static CreateKeyboardEventLikeWithKeyCode(event: IKeyboardEventLike, keyCode: number): IKeyboardEventLike
+   {
+    return {
```

Comment:
```suggestion
  private static CreateKeyboardEventLikeWithKeyCode(event: IKeyboardEventLike, keyCode: number): IKeyboardEventLike
  {
    return {
```

Look-see, there's been three spaces, now it's the correct two spaces to align with the function.